[
  {
    "DefinitionID": "oval:org.debian:def:157449709691417453117631642853543640184",
    "Title": "CVE-2017-5601 libarchive",
    "Description": "An error in the lha_read_file_header_1() function (archive_read_support_format_lha.c) in libarchive 3.2.2 allows remote attackers to trigger an out-of-bounds read memory access and subsequently cause a crash via a specially crafted archive.",
    "Advisory": {
      "Severity": "",
      "Cves": [
        {
          "CveID": "CVE-2017-5601",
          "Cvss2": "",
          "Cvss3": "",
          "Cwe": "",
          "Impact": "",
          "Href": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2017-5601",
          "Public": ""
        }
      ],
      "Bugzillas": [],
      "AffectedCPEList": [],
      "Issued": "1000-01-01T00:00:00Z",
      "Updated": "1000-01-01T00:00:00Z"
    },
    "Debian": {
      "MoreInfo": "",
      "Date": "1000-01-01T00:00:00Z"
    },
    "AffectedPacks": [
      {
        "Name": "libarchive",
        "Version": "3.2.1-6",
        "Arch": "",
        "NotFixedYet": false,
        "ModularityLabel": ""
      }
    ],
    "References": [
      {
        "Source": "CVE",
        "RefID": "CVE-2017-5601",
        "RefURL": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2017-5601"
      }
    ]
  }
]
